analyzere/base_resources.py

```diff
@@ -243,7 +243,7 @@ def upload_data(self, file_or_str):
 
         # Block until data has finished processing
         while True:
-            resp = self.status
+            resp = self.processing_status
```

Comment:
Yes it's necessary—if we don't either stop returning `status` on the root object, or change the name of the property here, the property will get overwritten when the object is retrieved (breaking large-file uploads).